SIL/script.js

```diff
@@ -0,0 +1,85 @@
+window.addEventListener("load", function(){
+  var load_screen = document.getElementById("load_screen");
+  document.body.removeChild(load_screen);
+});
+
+var request = new XMLHttpRequest();
+var areaInfoContainer = document.getElementById('area-Info');
+
+function clearAreaInfo() {
+  areaInfoContainer.innerHTML = "";
+};
+
+request.open('GET', 'https://code-your-future.github.io/api-demo/area/index.json', true);
+request.onload = function() {
+  if (request.status >= 200 && request.status < 400) {
+    // Convert the JSON response into an array.
+    var data = JSON.parse(request.responseText);
+    // Loop through the array.
+    for(var i = 0;i < data.data.length; i++) {
+      var element = document.createElement('div');
+      element.style.margin="3px 0";
+      element.style.width ="28%";
+      element.innerHTML = '<button id="btnid" onclick="areaData(\''+data.data[i]+'\')">'+data.data[i]+'</button>';
+      document.getElementsByTagName('body')[0].appendChild(element);
+    }
+  } else {
+    // We reached our target server, but it returned an error
+  }
+};
+
+
+request.send();
+
+request.onerror=function(){
+    console.log("Connection error");
+}
+
+// Get the data from the page.
+   var areaData= function(id) {
+    var ourRequest = new XMLHttpRequest();
+    ourRequest.open('GET', 'https://code-your-future.github.io/api-demo/area/' + id + '/index.json', true);
```

Comment:
I'd advise to use objects to store configurable data. For example, you can create an object at the top of the file with the api URLs, then use it when making the request.

```
const areaApiUrl = https://code-your-future.github.io/api-demo/area;

const apiUrls = {
  areaList: () => `${areaApiUrl}/index.json`,
  areaInfo: (id) => `${areaApiUrl}/${id}/index.json`
};

request.open('GET', apiUrls.areaList());
request.open('GET', apiUrls.areaInfo(areaId));
```